vdirsyncer/cli/tasks.py

```diff
@@ -142,11 +142,8 @@ async def metasync_collection(collection, general, *, connector: aiohttp.TCPConn
     try:
         cli_logger.info(f"Metasyncing {status_name}")
 
-        status = (
-            load_status(
-                general["status_path"], pair.name, collection.name, data_type="metadata"
-            )
-            or {}
+        status = load_status(
+            general["status_path"], pair.name, collection.name, data_type="metadata"
```

Comment:
**Nitpick**: can you include a trailing `,` here so that these arguments get auto-formatted into something easier to read?